assignment/index.html

```diff
@@ -62,10 +62,35 @@
 
       ===================== */
 
-      var jsonToCsv = function(json) { console.log(json); };
-
-
-      var addMarkers = function(map) {};
+      var header = [Object.keys(healthCenters[0])];
+      var TempArr = [Object.values(healthCenters)];
+      healthCentersArr = header.push(TempArr);
+      for (i=0; i < healthCentersArr.length; i++){console.log(healthCentersArr)};
+
+        var addIcon1 = L.icon({
+            iconUrl: 'https://upload.wikimedia.org/wikipedia/commons/3/3d/Blue_dot.png',
+            iconSize: [38, 95],
+            iconAnchor: [22, 94],
+            popupAnchor: [-3, -76],
+                  });
+        var addIcon2 =  L.icon({
+            iconUrl: 'https://www.pngarts.com/files/4/Dot-PNG-Pic.png',
+            iconSize: [38, 95],
+            iconAnchor: [22, 94],
+            popupAnchor: [-3, -76],
+                  });
+
+      var addMarkers1 = (map) => {return L.marker([obj[1],obj[0]]).bindPopuup(obj[3],{icon: addIcon1})};
```

Comment:
Also - I am not seeing any markers appear on the map. Were you able to view them?

![image](https://user-images.githubusercontent.com/8103418/52538562-33b37080-2d42-11e9-8934-6cbb425b3188.png)

Reply:
I think you very nearly have it! It is very likely one line that needs to be fixed. But just try to run it again and see if you can find it.

Reply:
Hi Ross, 
I fixed the code. It should be able to run this time!